datafusion/physical-optimizer/src/enforce_sorting/mod.rs

```diff
@@ -151,10 +165,51 @@ fn update_coalesce_ctx_children(
     };
 }
 
-/// The boolean flag `repartition_sorts` defined in the config indicates
-/// whether we elect to transform [`CoalescePartitionsExec`] + [`SortExec`] cascades
-/// into [`SortExec`] + [`SortPreservingMergeExec`] cascades, which enables us to
-/// perform sorting in parallel.
+/// If `repartition_sorts` is enabled,
+/// then transform [`CoalescePartitionsExec`] + [`SortExec`] cascades
+/// into [`SortExec`] + [`SortPreservingMergeExec`] cascades.
```

Comment:
I found cascades a bit confusing at first. Maybe something like this would be clearer:

```suggestion
/// then transform [`CoalescePartitionsExec`] + [`SortExec`]
/// into [`SortExec`] + [`SortPreservingMergeExec`] as illustrated below
```